test/test_site/_markbind/variables.md

```diff
@@ -7,3 +7,6 @@
 <span id="reference_level_2">{{reference_level_1}}</span>
 <span id="reference_level_3">{{reference_level_2}}</span>
 <span id="reference_level_4">{{reference_level_3}}</span>
+
+<span id="includedVariableGlobalOverridden">Included variable overriding page variables</span>

```

Comment:
`Included variable overriding page variables` -> `Global variable overriding page variables`?

Reply:
Also, use `kebab-case` for `id` (and `class`).